.travis.yml

```diff
@@ -1,29 +1,45 @@
+sudo: required
```

Comment:
Is there a point to do this? Switches automatically to trusty with `sudo: required`.